packages/langium/test/grammar/type-system/inferred-types.spec.ts

```diff
@@ -564,6 +564,20 @@ describeTypes('inferred types with common names and actions', `
 
 });
 
+describeTypes('inferred types with common names and actions', `
```

Comment:
I'd propose to name this test group "inferred types that are used by the grammar", because the ability to use inferred rule types at the grammar type level is checked here.

Reply:
Name was copied from the describe block above 😅